app/javascript/application.js

```diff
@@ -1,3 +1,13 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+import React from "react";
+import ReactDOM from "react-dom";
+import Greetings from "./components/Greetings";
+function App() {
```

Comment:
- [ ] It will be great to have a react-router set up in this component. this is a major requirement for this project to be approved. Kindly check below as stated in the project requirement ⬇️ 
![Screenshot 2023-05-04 at 12 23 41](https://user-images.githubusercontent.com/32361963/236164436-eb9a1b21-272c-48d6-9d41-6da28ee79151.png)